app.js

```diff
@@ -39,34 +43,30 @@ app.use(session(sessionParms));
 
 app.use(require("connect-flash")());
 
-app.set("view engine", "ejs");
+//body parser should be before passport
 app.use(require("body-parser").urlencoded({ extended: true}));
 
-// secret word handling
-// let secretWord = "syzygy";
+//initialize Passport
+const passport = require("passport");
```

Comment:
Up to you on organizing all parts of the feature together, but general convention is to put all imports at the top of the file. Just informational, it all runs the same! But helps to catch import errors early